service/src/main/java/uk/gov/hmcts/reform/fpl/service/payment/PaymentService.java

```diff
@@ -108,16 +98,8 @@ private CreditAccountPaymentRequest getCreditAccountPaymentRequest(Long caseId,
             .build();
     }
 
-    private void callPaymentsApi(CreditAccountPaymentRequest creditAccountPaymentRequest) {
-        try {
-            paymentApi.createCreditAccountPayment(requestData.authorisation(),
-                authTokenGenerator.generate(),
-                creditAccountPaymentRequest);
-        } catch (FeignException ex) {
-            log.error("Payments response error for {}\n\tstatus: {} => message: \"{}\"",
-                creditAccountPaymentRequest, ex.status(), ex.contentUTF8(), ex);
-
-            throw new PaymentsApiException(ex.contentUTF8(), ex);
-        }
+    // callPaymentsApi has been called from PaymentClient because of springboot limitations on retry mechanism
+    public void callPaymentsApi(CreditAccountPaymentRequest creditAccountPaymentRequest) {
+        paymentClient.callPaymentsApi(creditAccountPaymentRequest);
```

Comment:
we could call this paymentClient.callPaymentsApi directly in makePaymentForCaseOrders & makePaymentForC.. it would remove need for this method

Reply:
changed